[docs] Add status header for elastic exporter (#10314)
open-telemetry/opentelemetry-collector-contrib: exporter/elasticexporter/README.md

# Elastic native support for OpenTelemetry

| Status                   |                  |
| ------------------------ |------------------|

> | Stability                | [beta]           |

Reviewer: ```suggestion
| Stability                | [deprecated]           |
```

Since the Elastic 7.13 release, Elastic APM Server has supported OTLP directly. This `elastic` exporter was deprecated at that point. See https://www.elastic.co/guide/en/apm/guide/current/open-telemetry.html#open-telemetry-native

# Elastic native support for OpenTelemetry
# Deprecated Elastic Exporter

| Status                   |                 |
| ------------------------ |-----------------|
| Stability                | [deprecated]    |
| Supported pipeline types | trace, metrics  |
| Distributions            | [contrib]       |

> ℹ️ This exporter has been deprecated due to elastic supports native OTLP ingestion since version 7.13. This means you can use an [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter) to send data to Elastic, instead of the OpenTelemetry Collector Exporter for Elastic. **The `otlp` exporter is the recommended way to integrate the OpenTelemetry Collector to Elastic.**

Reviewer: ```suggestion
ℹ️ This exporter has been deprecated due to Elastic APM Server supporting native OTLP ingestion since version 7.13. This means you can use an [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter) to send data to Elastic APM Server, instead of the OpenTelemetry Collector Exporter for Elastic. **The `otlp` exporter is the recommended way to integrate the OpenTelemetry Collector to Elastic.**
```

        apm_server_url: "https://elasticapm.example.com"
        secret_token: "hunter2"
```

> [beta]:https://github.com/open-telemetry/opentelemetry-collector#beta

Reviewer: ```suggestion
[deprecated]:https://github.com/open-telemetry/opentelemetry-collector#deprecated
```
